go/glide_cluster_client.go

```diff
@@ -1216,16 +1208,16 @@ func (client *ClusterClient) ClientSetNameWithOptions(ctx context.Context,
 //	The name of the client connection as a string if a name is set, or nil if  no name is assigned.
 //
 // [valkey.io]: https://valkey.io/commands/client-getname/
-func (client *ClusterClient) ClientGetName(ctx context.Context) (models.ClusterValue[string], error) {
+func (client *ClusterClient) ClientGetName(ctx context.Context) (models.ClusterValue[models.Result[string]], error) {
```

Comment:
Should have a doc message
> The command will be routed to a random node

And this command (I mean command without route) never returns a multi-node value - only a single value. So you can simplify the return type.